plugins/src/main/java/org/smartregister/fhir/gateway/plugins/PermissionAccessChecker.java

```diff
@@ -201,12 +205,12 @@ private boolean checkIfRoleExists(String roleName, List<String> existingRoles) {
         return existingRoles.contains(roleName);
     }
 
-    private IGenericClient createFhirClientForR4(FhirContext fhirContext) {
+    public static IGenericClient createFhirClientForR4(FhirContext fhirContext) {
```

Comment:
I think we can move all these helper methods to the [Utils.java](https://github.com/onaio/fhir-gateway-extension/blob/main/plugins/src/main/java/org/smartregister/fhir/gateway/plugins/Utils.java) class